Test/Controller/ParamSpec.hs

```diff
@@ -141,32 +142,53 @@ tests = do
 
                 it "should accept floating point input" do
                     (readParameter @Double "1.2") `shouldBe` (Right 1.2)
-                    (readParameter @Float "1.2345679") `shouldBe` (Right 1.2345679)
-                
+                    (readParameter @Double "1.2345679") `shouldBe` (Right 1.2345679)
```

Comment:
While at it, I've fixed an apparent copy-paste error in `ParamReader Double` tests. Which reasonably speaking doesn't affect anything, but hey.

Reply:
Good catch :)